taxcalc/calcfunctions.py

```diff
@@ -1675,13 +2836,86 @@ def NonrefundableCredits(c05800, e07240, e07260, e07300, e07400,
 
     Parameters
     ----------
-    CTC_refundable: Whether the CTC is a refundable tax credit
```

Comment:
@amshoulders The `CTC_refundable` parameter needs to be added to the `Parameters` section.